server/verify_test.go

```diff
@@ -1068,10 +1068,14 @@ func TestVerifyAttestationWithSevSnp(t *testing.T) {
 			}
```

Comment:
Please add tests for the failure cases

Reply:
Added a new test `TestParseTEEAttestation` to cover both successful and failure cases.